gtsfm/averaging/rotation/rotation_averaging_base.py

```diff
@@ -14,14 +14,26 @@
 from gtsfm.common.pose_prior import PosePrior
 from gtsfm.evaluation.metrics import GtsfmMetric, GtsfmMetricsGroup
 
+from gtsfm.ui.gtsfm_process import GTSFMProcess, UiMetadata
 
-class RotationAveragingBase(metaclass=abc.ABCMeta):
+
+class RotationAveragingBase(GTSFMProcess):
     """Base class for rotation averaging.
 
     This class generates global rotation estimates from the pairwise relative
     rotations.
     """
 
+    def get_ui_metadata() -> UiMetadata:
+        """Returns data needed to display this process in the process graph."""
```

Comment:
nit: I would prefer we say something a bit more informative like
"""Returns data necessary to display node and edge information for the current process in the process graph."""

Reply:
changed to: 
> """Returns data needed to display node and edge info for this process in the process graph."""